zigzag/zigzag_test_log.py

```diff
@@ -303,3 +303,27 @@ def _get_test_run_failure_output_field_id(cls, mediator):
             cls._test_run_failure_output_field_id = \
                 UtilityFacade(mediator).find_custom_field_id_by_label('Failure Output', 'test-runs')
         return cls._test_run_failure_output_field_id
+
+    @classmethod
+    def _get_fields(cls, mediator):
+        """Gets the fields from this class
+
+        Args:
+            mediator (ZigZag): The ZigZag mediator
+
+        Returns:
+            dict(str: ZigZagTestLogField)
+        """
+
+        if cls._fields == 0:
+            cls._fields = {}
+            for prop, value in list(mediator.testsuite_props.items()):
```

Comment:
Using `list` is unnecessary. Can be rewritten as:

```python
for prop, value in mediator.testsuite_props.items():
```

Reply:
I read that this was the way to do it so that it worked in 2.7 & 3